plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.test.ts

```diff
@@ -48,7 +48,9 @@ describe('fetch:template', () => {
     ActionContext<FetchTemplateInput>['createTemporaryDirectory']
   > = jest.fn(() =>
     Promise.resolve(
-      `${workspacePath}/${createTemporaryDirectory.mock.calls.length}`,
+      joinPath(
```

Comment:
Think this should be:

`joinPath(workspacePath, createTemporaryDirectory.mock.calls.length)`

Reply:
Yep, my bad - discussed in [this thread](https://github.com/backstage/backstage/pull/6322#discussion_r665161738) too :)